tests/integration_tests/test_fastai.py

```diff
@@ -0,0 +1,84 @@
+from functools import partial
+import typing  # NOQA
+
+from fastai.basic_data import DataBunch
+from fastai.basic_train import Learner
+from fastai.metrics import accuracy
+
```

Comment:
[nits] Please remove this black line to follow [PEP-8](https://www.python.org/dev/peps/pep-0008/#imports) convention (see below).

```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports.
(from "Imports" section of PEP-8)
```

Reply:
Do you think the below OK?

```python
from functools import partial
import typing  # NOQA

from fastai.basic_data import DataBunch
from fastai.basic_train import Learner
from fastai.metrics import accuracy
import numpy as np
import pytest
import torch.nn as nn
from torch.utils.data import Dataset

import optuna
from optuna.integration import FastAIPruningCallback
from optuna.testing.integration import DeterministicPruner
```

Reply:
Yes. I think that it's the typical style used in Optuna code.